<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageBooksView</class>
 <widget class="QWidget" name="manageBooksView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="rightMargin">
       <number>222</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_home_page">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Home Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_your_account">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Your Account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_manage_books">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Manage Books</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8"/>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>707</width>
         <height>390</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_return_book">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Return a book</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>351</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>8</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Provide PESEL of a person returning a book along with BookID being returned</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_book_id">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>BookID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_pesel_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>PESEL</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_remove_book">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>250</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Remove a book</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_book_id_rem">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>BookID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_quantity">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>Quantity</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>8</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Provide BookID and Quantity of books to be removed</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>10</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>8</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Provide information on book that you're adding</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_book_title">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>30</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>Title</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_author_first_name">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>70</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>Author's First Name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_author_last_name">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>110</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>Author's Last Name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_quantity_add">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>150</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>Quantity</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_add_book">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>190</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Add a book</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
